rosdep/python.yaml

```diff
@@ -6012,10 +6012,20 @@ python3-flask:
   fedora: [python3-flask]
   gentoo: [dev-python/flask]
   ubuntu: [python3-flask]
+python3-flask-cors:
+  debian: [python3-flask-cors]
+  ubuntu: [python3-flask-cors]
 python3-flask-restplus-pip:
   ubuntu:
     pip:
       packages: [flask-restplus]
+python3-flask-socketio:
+  debian:
+    pip:
+      packages: [flask-socketio]
+  ubuntu:
+    pip:
+      packages: [flask-socketio]
```

Comment:
This is not a parallel implelementation and will conflict with a system install. 5.0 is already packaged for hirsute: https://packages.ubuntu.com/source/hirsute/flask-socketio

Reply:
Make sense, fixed. https://github.com/ros/rosdistro/pull/29086/commits/c9f227ff7f785fdf657adcf9c5014939ce98b341